pokete_classes/landscape.py

```diff
@@ -1,4 +1,5 @@
 import random
+import logging
```

Comment:
`logging` is never used, so why was it imported?

```suggestion
```

Reply:
For debugging.